xml/deploy_cephadm.xml

```diff
@@ -1089,6 +1116,71 @@ ses-min4
      </listitem>
     </itemizedlist>
    </sect3>
+   <sect3 xml:id="deploy-cephadm-day2-service-mds">
+    <title>Deploy &mds;s</title>
+    <para>
+     To use &cephfs;, you need to deploy at least one &mds; (MDS) service. You
+     can either deploy MDS's without &cephfs;, or create &cephfs; and deploy an
+     MDS automatically.
+    </para>
+    <para>
+     To deploy &mds; without &cephfs;, run:
+    </para>
+<screen>&prompt.cephuser;ceph orch apply mds <replaceable>CEPH_FS_NAME</replaceable> <replaceable>HOST1</replaceable>,<replaceable>HOST2</replaceable>,...</screen>
+    <para>
+     To create &cephfs; and deploy &mds; automatically, run:
```

Comment:
I am not sure -- is this one MDS or several? 

If one, then either "deploy _the_ MDS" or "deploy _an_ MDS".